src/sedml/SedFitExperiment.cpp

```diff
@@ -144,16 +144,6 @@ SedFitExperiment::~SedFitExperiment()
 }
 
```

Comment:
you are not changing the id / name allowed members here, cause it is l1v4? Would that ever cause an issue?

Reply:
Right; I assumed that anything that was added in l1v4 is already handled and not written out in conversions to l1v3.  (If not, I guess that's a separate issue we should file?)

Reply:
ok